README.md

```diff
@@ -60,6 +60,19 @@ There are many ways to contribute to HEIR:
     to alert us of papers that should be investigated and incorporated into
     HEIR.
 
+## Citations
+
+The HEIR project can be cited in in academic work through following entry:
+
+```text
+@Misc{HEIR,
+  title={{HEIR: Homomorphic Encryption Intermediate Representation}},
+  author={Google LLC},
```

Comment:
The convention for open-source projects is usually something like "<project> contributors"
```suggestion
  author={HEIR Contributors},
```